lib/fuse.c

```diff
@@ -4702,6 +4705,12 @@ struct fuse *fuse_new_31(struct fuse_args *args,
 		}
 	}
 
+
+	if (f->conf.max_idle_threads == -1) {
+		/* If the number of idle threads was not set, default to 10. */
+		f->conf.max_idle_threads = 10;
+	}
+
```

Comment:
Wouldn't you achieve exactly that if you set the default to 10 when initializing the structure (before the command line is parsed), and get rid of this `if` clause completely?